INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 70, in get_plane
    constellation =  ephem.constellation(mars)
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 25, in get_plane
    mars=ephem.Mars(a)
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 73, in get_plane
    update.message.reply_text((ephem.constellation(mars)))
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 71, in get_plane
    epherrr=ephem.constellation(mars)
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 71, in get_plane
    epherrr=ephem.constellation(mars)
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 16, in get_plane
    user_text = str(update.message.text)
AttributeError: 'NoneType' object has no attribute 'text'
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 71, in get_plane
    epherrr=mars.constellation(a)
AttributeError: 'Sun' object has no attribute 'constellation'
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 71, in get_plane
    epherrr=constellation(mars)
UnboundLocalError: local variable 'constellation' referenced before assignment
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 68, in get_plane
    mars=ephem.Sun(a)
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 68, in get_plane
    mars=ephem.Sun(a)
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 68, in get_plane
    mars=ephem.Sun()
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 68, in get_plane
    mars=ephem.Sun()
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 68, in get_plane
    mars=ephem.Sun()
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 68, in get_plane
    mars=ephem.Sun()
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 68, in get_plane
    mars1=ephem.Sun()
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 56, in get_plane
    mars=ephem.Neptune(a)
NameError: name 'ephem' is not defined
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 16, in get_plane
    user_text = str(update.message.text)
AttributeError: 'NoneType' object has no attribute 'text'
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 25, in get_plane
    mars=ephem.Mars(a)
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
ERROR:telegram.ext.dispatcher:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 425, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/vartan/.local/lib/python3.8/site-packages/telegram/ext/handler.py", line 145, in handle_update
    return self.callback(update, context)
  File "bot.py", line 61, in get_plane
    mars=ephem.Pluto(a)
NameError: name 'ephem' is not defined
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
INFO:root:Бот стартовал
INFO:apscheduler.scheduler:Scheduler started
